### PR TITLE
add cromwell recipe

### PR DESCRIPTION
**Pull request by rhpvorderman**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

I am willing to maintain this recipe as long as my employer uses cromwell. Which we plan to do for the foreseeable future.

@conda-forge/help-java Can you have a look? As this is my first package in conda-forge.

### CODE REVIEW

**Comment by rhpvorderman** [comment]
Cromwell is listed to be moved in this issue on bioconda: https://github.com/bioconda/bioconda-recipes/issues/13964

**Comment by rhpvorderman** [comment]
Added @micknudsen as maintainer (https://github.com/bioconda/bioconda-recipes/pull/15174#issuecomment-490382877).

**Comment by rhpvorderman** [comment]
@conda-forge/help-java 
The proposed package has now four maintainers and is ready for merging. Looking forward to your feedback! Thanks!

**Comment by rhpvorderman** [comment]
@conda-forge/help-java
This package has now four maintainers and is ready for merging. I am looking forward to your feedback! Thanks!

File: recipes/cromwell/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    sha256: {{ sha256_womtool }}
+    folder: womtool
+
+build:
+  # Cannot build on Windows because findutils is not available
+  skip: true  # [win]
+  number: 0
+
+requirements:
+  host: []
```
Comment (chrisburr) [inline recipes/cromwell/meta.yaml]:
```suggestion
```

File: recipes/cromwell/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    - findutils
+
+test:
+  commands:
+    - cromwell --version 2>&1 | grep "cromwell"
+    - womtool --help 2>&1 | grep "Usage"
+
+about:
+  home: https://github.com/broadinstitute/cromwell
+  license: "BSD"
```
Comment (chrisburr) [inline recipes/cromwell/meta.yaml]:
```suggestion
  license: BSD-3-Clause
  license_family: BSD
```

File: recipes/cromwell/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+{% set sha256_cromwell = "8e1f9c8777bb3b860fd1bf6b6dc05a51145889620f0510cb2fa89aa7e1dfd3ec" %}
+{% set sha256_womtool = "1904ebc87c605b4330ddd88770b522104d0bd506be2b9bcfe8424e90d38e4fb9" %}
+
+package:
+  name: cromwell
+  version: 0.{{ version }}
+
+source:
+  - url: https://github.com/broadinstitute/cromwell/releases/download/{{ version }}/cromwell-{{ version }}.jar
+    sha256: {{ sha256_cromwell }}
```
Comment (chrisburr) [inline recipes/cromwell/meta.yaml]: If you move the hash here and delete the variable I think the bot will stand a better chance of automatically updating:
```suggestion
    sha256: 8e1f9c8777bb3b860fd1bf6b6dc05a51145889620f0510cb2fa89aa7e1dfd3ec
```

File: recipes/cromwell/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+    folder: cromwell
+  - url: https://github.com/broadinstitute/cromwell/releases/download/{{ version }}/womtool-{{ version }}.jar
+    sha256: {{ sha256_womtool }}
```
Comment (chrisburr) [inline recipes/cromwell/meta.yaml]: Same
```suggestion
    sha256: 1904ebc87c605b4330ddd88770b522104d0bd506be2b9bcfe8424e90d38e4fb9
```

File: recipes/cromwell/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,45 @@
+{% set version = "40" %}
+{% set sha256_cromwell = "8e1f9c8777bb3b860fd1bf6b6dc05a51145889620f0510cb2fa89aa7e1dfd3ec" %}
```
Comment (chrisburr) [inline recipes/cromwell/meta.yaml]:
```suggestion
```

File: recipes/cromwell/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+{% set sha256_womtool = "1904ebc87c605b4330ddd88770b522104d0bd506be2b9bcfe8424e90d38e4fb9" %}
```
Comment (chrisburr) [inline recipes/cromwell/meta.yaml]:
```suggestion
```

File: recipes/cromwell/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+
+package:
+  name: cromwell
+  version: 0.{{ version }}
```
Comment (chrisburr) [inline recipes/cromwell/meta.yaml]: Is there a reason for prepending `0` here?

Comment (rhpvorderman) [reply recipes/cromwell/meta.yaml]: History reasons. This package is moved from Bioconda. The package has been in bioconda for quite some time. I don't know about the motivation of the original writer of the package. 
The move to conda-forge is a nice opportunity to bring the version in line with the actual version number.

**Comment by rhpvorderman** [comment]
@chrisburr Thanks for reviewing! I have updated the recipe.

**Pull request merged**